base/operators.jl

```diff
@@ -827,17 +827,25 @@ julia> [1:5;] |> x->x.^2 |> sum |> inv
 Compose functions: i.e. `(f ∘ g)(args...)` means `f(g(args...))`. The `∘` symbol can be
 entered in the Julia REPL (and most editors, appropriately configured) by typing `\\circ<tab>`.
 
+Function composition also works in prefix form: `∘(f,g)` is the same as `f ∘ g`.
+The prefix form also supports multiple function composition: `∘(f,g,h) = f ∘ g ∘ h`
+and splatting `∘(fs...)` for an iterable.
+
 # Examples
 ```jldoctest
 julia> map(uppercase∘first, ["apple", "banana", "carrot"])
 3-element Array{Char,1}:
  'A'
  'B'
  'C'
+
+julia> fs = (x -> 2x, x -> x/2, x -> x-1, x -> x+1); f = ∘(fs...); f(3) == 3
+true
 ```
 """
 ∘(f, g) = (x...)->f(g(x...))
-
+∘(f, g, h) = ∘(f ∘ g, h)
+∘(f, g, h, is...) = ∘(∘(f, g), h, is...)
```

Comment:
Can't this be done with just a single method?
```jl
∘(f, g, h...) = ∘(f ∘ g, h...)
```
Edit: confirmed, it can.

Reply:
I had been hoping for a one-liner and I had tried that in a test sandbox but I kept getting infinite recursion, but I see now that I had errors in the way I was using `Base.:(∘)`
It is much more elegant as a single line so I have updated the PR to use that version.  Thanks!